src/main/java/com/genesys/workspace/models/DNumber.java

```diff
@@ -1,14 +1,14 @@
 package com.genesys.workspace.models;
 
-public class Dn {
+public class DNumber {
```

Comment:
Dn would be the preferred naming for this model - can we preserve it?

Reply:
sure, I rename it